servicepulse/intro-pending-retries.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Pending Retries Message Management in ServicePulse
```

Comment:
I think if we have add a brief introduction in the [intro to failed messages page](http://docs.particular.net/servicepulse/intro-failed-messages#failed-message-retry) about retries and point the users to this page, then this page can focus on retrying a message including what happens to the pending retries. Thoughts?